Demo/Controllers/LoginViewController.swift

```diff
@@ -14,13 +14,31 @@ class LoginViewController: UIViewController {
     @IBOutlet weak var activityIndicator: UIActivityIndicatorView!
     @IBOutlet weak var usernameTextField: UITextField!
     
+    // MARK: - Properties
+    
+    private var chatkit: Chatkit?
+    
     // MARK: - View lifecycle
     
     override func viewDidAppear(_ animated: Bool) {
         super.viewDidAppear(animated)
         self.displayKeyboard()
     }
     
+    // MARK: - Navigation
+    
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        super.prepare(for: segue, sender: sender)
+
+        if segue.identifier == LoginViewController.loginSegue {
+            guard let roomListViewController = segue.destination as? RoomListViewController else {
```

Comment:
This code doesn't work at present because `segue.destination` is a `UINavigationController`.  This works instead...
```suggestion
        if segue.identifier == LoginViewController.loginSegue {
            guard let navigationController = segue.destination as? UINavigationController,
                let roomListViewController = navigationController.viewControllers[0] as? RoomListViewController else {
```